Code/GraphMol/RascalMCES/Wrap/testRascalMCES.py

```diff
@@ -92,17 +92,6 @@ def test4(self):
     results = rdRascalMCES.FindMCES(mol1, mol2, opts)
     self.assertEqual(len(results), 1)
 
-  def test5(self):
```

Comment:
Why was this test removed?

Reply:
Good catch.  During the "thrashing about trying to solve the problem" phase, this test took ridiculous amounts of time, so I took it out and failed to put it back in again when things had settled down.  The result has changed.  The new result is preferred because it has 1 more atom than the previous solution, even if it has 1 less bond.  The hits are sorted by number of atoms then number of bonds.